### PR TITLE
add a blog for Trying Out An Upcoming Security Feature for WildFly on OpenShift

### PR DESCRIPTION
**Pull request by PrarthonaPaul**

### CODE REVIEW

File: _posts/2024-03-12-in-progress-wildfly-feature-on-openshift.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Trying Out An Upcoming Security Feature for WildFly on OpenShift'
+date: 2024-03-12
+tags: oidc openshift authentication scopes
+synopsis: How to deploy a wildfly app on OpenShift using an upcoming feature that introduces the ability to add additional scope values when securing applications using OpenID Connect.
```
Comment (fjuma) [inline _posts/2024-03-12-in-progress-wildfly-feature-on-openshift.adoc:6]: s/wildfly/WildFly

**Comment by bstansberry** [comment]
@PrarthonaPaul  FYI re https://lists.jboss.org/archives/list/wildfly-dev@lists.jboss.org/thread/QECMN7WAAJ4WL5E5FUYCT7IRKEI4LDOO/

It seems a couple lines of summary content a la https://github.com/wildfly/wildfly.org/pull/569/files#diff-9c25ff43f57cb32da95825c57ea257d0c796fb255b19cebc171e490f457ff4caR10-R14 is a best practice.

File: _posts/2024-03-12-in-progress-wildfly-feature-on-openshift.adoc (hunk 1 above)
Comment (PrarthonaPaul) [reply _posts/2024-03-12-in-progress-wildfly-feature-on-openshift.adoc]: fixed!

**Comment by PrarthonaPaul** [comment]
> @PrarthonaPaul FYI re https://lists.jboss.org/archives/list/wildfly-dev@lists.jboss.org/thread/QECMN7WAAJ4WL5E5FUYCT7IRKEI4LDOO/
> 
> It seems a couple lines of summary content a la https://github.com/wildfly/wildfly.org/pull/569/files#diff-9c25ff43f57cb32da95825c57ea257d0c796fb255b19cebc171e490f457ff4caR10-R14 is a best practice.

Hi @bstansberry 
Thanks for pointing that out. 
I have added the intro to the end of the post. 
I have some hyperlinks there. Hopefully, those render okay. If not, then I can get rid of them.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**